net.lutris.Lutris.yml

```diff
@@ -716,3 +716,22 @@ modules:
     sources:
       - type: file
         path: ld.so.conf
+
+  - name: perl
+    no-autogen: true
+    config-opts:
+      - -des
+      - -Duseshrplib
+    build-options:
+      - cflags: -fPIC
+      - ldflags: -fpic
+    sources:
+      - type: archive
+        url: "https://www.cpan.org/src/5.0/perl-5.32.1.tar.gz"
+        sha256: 03b693901cd8ae807231b1787798cf1f2e0b8a56218d07b7da44f784a7caeb2c
+      - type: script
+        dest-filename: configure
+        commands:
+          - exec ./configure.gnu $@
+    post-install:
+      - chmod -R 755 /app/lib/perl5/5.32.1/x86_64-linux
```

Comment:
If someone is able to build without this line please tell me how :)

Reply:
Check https://github.com/flathub/io.github.Hexchat.Plugin.Perl/blob/master/io.github.Hexchat.Plugin.Perl.json. More examples at https://github.com/search?q=org%3Aflathub+perl&type=code.

Reply:
I copied from there anyhow :+1: 
But I get permission errors if I don't have the chmod line

Maybe something is configured wrong on my end.